public/admin/config.yml

```diff
@@ -121,41 +121,52 @@ meta_config: &meta_config
     - name: "og"
       label: "Open Graph Metadata"
       widget: "object"
+      required: false
       collapsed: true
+      hint: "Expand for more information on default values."
       fields:
         - name: "title"
           label: "Open Graph Title"
           widget: "string"
           required: false
+          hint: "If left empty, the SEO meta title will be used."
         - name: "image"
           label: "Open Graph Image"
           widget: "image"
           allow_multiple: false
           choose_url: false
           required: false
+          hint: "If left empty, the page header image will be used. If no header image is present, a default homepage image will be used."
     - name: "twitter"
       label: "Twitter Metadata"
       widget: "object"
+      hint: "Expand for more information on default values."
+      required: false
       collapsed: true
       fields:
-        - name: "title"
-          label: "Twitter Title"
-          widget: "string"
-          default: "summary"
-          required: false
         - name: "card"
           label: "Twitter Card"
-          widget: "string"
-          default: "summary"
+          widget: "select"
+          options:
+            - label: "Summary Card"
+              value: "summary"
+            - label: "Summary with Large Image"
+              value: "summary_large_image"
+            - label: "Player Card"
+              value: "player"
```

Comment:
@barbaraperic @CharlyMartin I’ve updated this to use a select widget.

Would we want to dynamically retrieve the Twitter Card values from the CMS JSON and feed them into `const TwitterCardType = z.enum(['summary', 'summary_large_image', 'player'])`?

The alternative is to leave it hardcoded, but the risk is that if someone changes the values in the CMS, Zod will only throw an error once a data entry with a new or incorrect value is added.

Reply:
I would like to get this merged - So happy to discuss this and a ticket for it.